ignite/src/test/resources/skipTests

```diff
@@ -47,4 +47,7 @@ org.hibernate.ogm.backendtck.associations.collection.unidirectional.CollectionUn
 org.hibernate.ogm.backendtck.associations.manytoone.ManyToOneTest#testUnidirectionalManyToOne
 org.hibernate.ogm.backendtck.jpa.JPAResourceLocalTest#testBootstrapAndCRUD
 
+//wait answer from experts
+org.hibernate.ogm.backendtck.callbacks.PostUpdateTest
+
```

Comment:
PostUpdate should work now can you test it?

Reply:
Ok. I will retest after update core.

Reply:
yes. it works. thanks! What did you do for it ? Please, open the secret :)

Reply:
I just simplify the test, I don't remember exactly